Gutenberg/cocoapods_helpers.rb

```diff
@@ -126,6 +126,11 @@ def gutenberg_post_install(installer:)
   react_native_post_install(installer, react_native_path)
 end
 
+def gutenberg_post_integrate
+  # If the this workaround runs in the post_install step, the changes it makes get overridden somehow.
+  workaround_broken_search_paths
+end
```

Comment:
This tells us something interesting about the possible culprit.

- Has something changed in the project integration phase in CocoaPods that could be formatting the path in that incorrect way?
- Or, is there something that React Native is capable of running at that time, even though we don't call it explicitly?